src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -3112,15 +3125,18 @@ public enum LegendsAwakened implements LogicCardInfo {
             energyCost ()
             attackRequirement {}
```

Comment:
I'd probably put a `assert my.bench` here as without a benched Pokémon this attack would have no effect.